data/faq.md

```diff
@@ -12,79 +12,42 @@ To stay up to date on the latest information on sessions, speakers, and activiti
 
 During two conference days attendees will be able to attend sessions and hands-on workshops, chat with experts and attendees at coffee breaks and after party, and more.
 
+### Refunds
+
+Refunds may be requested up to two weeks prior to the event. To request a refund, email [info@libertyjs.com](mailto:info@libertyjs.com) and include your name and order number.
+No refunds are available for cancellations made within 14 days of the event.
```

Comment:
note: everything below this is commented out and does not appear on the page. I removed a few sections that obv dont apply to us